src/views/List.jsx

```diff
@@ -7,11 +7,9 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item) => {
+					return <ListItem key={item.id} name={item.name} />;
+				})}
```

Comment:
Minor suggestion (either for this PR or for the next time when it arises) since its a one line return:
```
{data.map((item) => {
	return <ListItem key={item.id} name={item.name} />;
})}
```

to

```
{data.map((item) => <ListItem key={item.id} name={item.name} />;)}

Reply:
Good call with implicit return!

Reply:
`return` removed to make this an implicit return and pushed up to this branch.